croncat/src/config/mod.rs

```diff
@@ -1,28 +1,83 @@
+use chain_registry::{cache::RegistryCache, paths::Tag};
 /// TODO: Move to chain registry
 /// Right now juno testnet missing grpc's, so we keeping it like `cosm-orc`'s chain config
 use color_eyre::Report;
 use config::Config;
+use cosmos_sdk_proto::tendermint::p2p::NetAddress;
 use serde::{Deserialize, Serialize};
-
 const CONFIG_FILE: &str = "config.yaml";
+const CONFIG_FILE_OVERRIDE: &str = "config.override.yaml";
```

Comment:
So `config.override.yaml` is for testnet right?

Reply:
> 

yes